internal/cli/config_file.go

```diff
@@ -27,17 +27,18 @@ type SuiteConfigResults struct {
 }
 
 type SuiteConfigRetries struct {
-	Attempts                  int
-	Command                   string
-	FailFast                  bool     `yaml:"fail-fast"`
-	FailOnMisconfiguration    bool     `yaml:"fail-on-misconfiguration"`
-	FlakyAttempts             int      `yaml:"flaky-attempts"`
-	MaxTests                  string   `yaml:"max-tests"`
-	MaxTestsLegacyName        string   `yaml:"maxtests"`
-	PostRetryCommands         []string `yaml:"post-retry-commands"`
-	PreRetryCommands          []string `yaml:"pre-retry-commands"`
-	IntermediateArtifactsPath string   `yaml:"intermediate-artifacts-path"`
-	AdditionalArtifactPaths   []string `yaml:"additional-artifact-paths"`
+	Attempts                   int
+	Command                    string
+	FailFast                   bool     `yaml:"fail-fast"`
+	FailOnMisconfiguration     bool     `yaml:"fail-on-misconfiguration"`
+	FlakyAttempts              int      `yaml:"flaky-attempts"`
+	MaxTests                   string   `yaml:"max-tests"`
+	MaxTestsLegacyName         string   `yaml:"maxtests"`
+	PostRetryCommands          []string `yaml:"post-retry-commands"`
+	PreRetryCommands           []string `yaml:"pre-retry-commands"`
+	IntermediateArtifactsPath  string   `yaml:"intermediate-artifacts-path"`
+	AdditionalArtifactPaths    []string `yaml:"additional-artifact-paths"`
+	SkipQuarantinedTestRetries bool     `yaml:"skip-quarantined-test-retries"`
```

Comment:
How do you feel about `QuarantinedAttempts` to mirror `FlakyAttempts` (I don't have a strong preference, just thinking - maybe Tommy/others feel differently)

Reply:
So, you'd set it to `0` to make it not retry quarantined tests, for example

Reply:
I like the symmetry of that.

Reply:
I like it too, updated both this PR and the corresponding docs PR https://github.com/rwx-cloud/docs/pull/568